index.html

```diff
@@ -78,7 +78,15 @@
 var channel = 'bad-attitude';
 
 function clean(text) {
-  return (''+text).trim().replace( /[<>]/g, '' );
+  var i = 0, m;
+  text = text.replace(/&/g, '&amp;').replace(/</g, '&gt;');
```

Comment:
* `<` is `&lt;`, right?  
* Why not replace both sides of the brackets?
* If you're going to fix these things here, you should probably not delete angle brackets above.